paper.md

```diff
@@ -0,0 +1,159 @@
+
+## Additional code health tools
+
+The LF-PPT supports many more features than those listed here including static code analysis, performance benchmarking, code testing coverage and more. For a complete list, please see the documentation at http://lincc-ppt.readthedocs.io/.
+
+The most notable exclusion from the LF-PPT is code. Aside from a few optional stub source and test files, the LF-PPT allows the user to focus on scientific code, while supporting them with the industry best practices in maintainable software engineering.
+
+# Usage
+
+## General usage
+
+The LINCC-Frameworks Python Project Template automates the setup of the above processes for Python projects hosted on GitHub. The only direct dependency is copier[^7] [@copier], which is used as the engine to generate new projects from the LF-PPT with a specific directory structure, the requested configuration and stub files. To hydrate a new or existing project with the LF-PPT via copier the user calls:
+
+```
+copier copy gh:lincc-frameworks/python-project-template <new/project/directory>
+```
+
+A questionnaire is presented to configure the project and establish the various features of the template to include. After the directory structure and files have been generated, the user should run the included initialization script to configure the local git repository and install the new package in the virtual environment:
+
+```
+cd <new/project/directory>
+bash .initialize-new-project.sh
+```
+
+The process is designed so as not to require significant time, and, thus, if the user is unhappy with the generated project, they can simply delete and recreate it. Depending on the options selected, some additional configuration may be required, such as registering with ReadTheDocs or PyPI. To assist the user, a customized, post-creation checklist is generated.
+
+## Existing projects
+
+In addition to creating new projects from scratch, the LF-PPT can be applied to existing projects to incorporate the features it provides. Often a more established project will require more effort to apply the LF-PPT. However, LINCC-Frameworks has had success applying the template to multiple legacy projects. A collection of tips for applying the template to existing projects is included in the documentation http://lincc-ppt.readthedocs.io/.
+
+## Updating projects
+
+As best practices evolve and new tools are introduced, the LF-PPT will incorporate those into the template. Updates to the template can be applied to existing projects with minimal effort, allowing the users to focus on science and not software maintenance.
```

Comment:
Is this as configurable as initial application of the template, where they get to decide which code health features they want to use?  If so, might be worth mentioning this configurability of updates.

Reply:
Yes, you're correct, `copier update` does allow the user to modify the initial choices. I'll add a line about that.

Reply:
Added this line after the :

"Similarly, if the user determines that different template options would better suit the project as it evolves, the same command can be used to review and change the initial responses to the questionnaire and apply those changes to the project."